script/deploy/single/07_TellerSetup.s.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import {RolesAuthority} from "@solmate/auth/authorities/RolesAuthority.sol";
+import {ManagerWithMerkleVerification} from "./../../../src/base/Roles/ManagerWithMerkleVerification.sol";
+import {BoringVault} from "./../../../src/base/BoringVault.sol";
+import {TellerWithMultiAssetSupport} from "./../../../src/base/Roles/TellerWithMultiAssetSupport.sol";
+import {AccountantWithRateProviders} from "./../../../src/base/Roles/AccountantWithRateProviders.sol";
+import {BaseScript} from "../../Base.s.sol";
+import {ConfigReader} from "../../ConfigReader.s.sol";
+import {CrossChainTellerBase} from "../../../src/base/Roles/CrossChain/CrossChainTellerBase.sol";
+import {ERC20} from "@solmate/tokens/ERC20.sol";
+
+contract TellerSetup is BaseScript {
+
+    function run() public virtual {
+        deploy(getConfig());
+    }
+
+    function deploy(ConfigReader.Config memory config) public virtual override broadcast returns(address){
+        TellerWithMultiAssetSupport teller = TellerWithMultiAssetSupport(config.teller);
+
+        teller.addAsset(ERC20(config.base));
+        AccountantWithRateProviders(teller.accountant()).setRateProviderData(ERC20(config.base), true, config.rateProvider);
```

Comment:
Thinking about how to simplify the process of initializing multiple deposit assets through the script. One way would be:

1. Deployer specifies an array of deposit assets in the `mydeploy.json` file under a "depositAssets" field. 
2. Deployer specifies a json mapping of chain -> deposit asset address -> rate provider. 
    - The rate providers for each chain must all be deployed in a separate step. (When deploying the vault on mainnet, we would use the same contracts and not deploy new rate providers. But the addresses would be saved to the config).
3. In the code above, the script iterates through the deposit assets, then reads the necessary rate provider address, then calls `teller.addAsset` for each.